### PR TITLE
Proofreading

### PR DESCRIPTION
**Pull request by Okeanos**
As requested in #514 I extracted the styling/grammar fixes into a separate PR and took the time to go over the other files as well (modern IDEs with text inspections really are awesome 🤯).

If there's anything I should change (back), please let me know and I'll have a look.

### CODE REVIEW

File: modules/ROOT/pages/running-containers.adoc [hunk 1]
```diff
@@ -34,7 +34,7 @@ systemd:
 
 === Running etcd
 
-etcd is not shipped as part of Fedora CoreOS. To use it, run it as a container, as shown below.
+https://etcd.io[etcd] is not shipped as part of Fedora CoreOS.To use it, run it as a container, as shown below.
```
Comment (bgilbert) [inline modules/ROOT/pages/running-containers.adoc:37]:
```suggestion
https://etcd.io[etcd] is not shipped as part of Fedora CoreOS. To use it, run it as a container, as shown below.
```

File: modules/ROOT/pages/running-containers.adoc [hunk 2]
```diff
@@ -1,7 +1,7 @@
 = Running Containers
 
 == Introduction
-Fedora CoreOS ships with both docker and podman installed. This page explains how to use systemd units to start and stop containers with podman.
+Fedora CoreOS ships with both https://www.docker.com[docker] and https://podman.io[podman] installed. This page explains how to use systemd units to start and stop containers with podman.
```
Comment (bgilbert) [inline modules/ROOT/pages/running-containers.adoc:4]: It ships with `docker` but not Docker.  Specifically, it ships with [Moby](https://mobyproject.org/), Docker's upstream.

**Review by bgilbert — COMMENTED** [review]
Really nice cleanup, thanks!

File: modules/ROOT/pages/running-containers.adoc (hunk 2 above)
Comment (Okeanos) [reply modules/ROOT/pages/running-containers.adoc]: Added a small insert explaining this.

**Review by bgilbert — APPROVED** [review]
Looks great, thank you!

**Pull request merged**